db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml

```diff
@@ -8690,42 +8673,36 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
 
             <msg>
                 <name>L001</name>
-                <text>------------  BEGIN ERROR MESSAGE -------------
-</text>
+                <text>------------ BEGIN ERROR MESSAGE -------------</text>
             </msg>
 
             <msg>
                 <name>L002</name>
-                <text>------------  END ERROR MESSAGE -------------
-</text>
+                <text>------------ END ERROR MESSAGE -------------</text>
             </msg>
 
             <msg>
                 <name>L003</name>
                 <text>
-------------  BEGIN SHUTDOWN ERROR STACK -------------
-</text>
+                    ------------ BEGIN SHUTDOWN ERROR STACK -------------</text>
```

Comment:
i guess it is intended you added 20 spaces in front of the message? (just checking)

Reply:
Actually not, this was part of IntelliJ's reformatting the file, I restored it